test/functional/legacy/045_folding_spec.lua

```diff
@@ -33,24 +37,89 @@ describe('folding', function()
     execute('call append("$", getline("."))')
     feed('zc')
     execute('call append("$", getline(foldclosed(".")))')
-    -- Test folding with markers.
-    execute('set fdm=marker fdl=1 fdc=3')
-    execute('/^5')
-    execute('call append("$", "marker " . foldlevel("."))')
+
+    expect([[
+      manual 1 aa
+      -1
+      3 cc
+      1 aa]], true)
+  end)
+
+  it("with marker foldmethod", function()
```

Comment:
For clarity, brevity, and more recognizable as "this is a Vim option":

    it("foldmethod=marker", function()